### ISSUE TITLE
Ignores files with object destructuring 

### ISSUE THREAD

**Issue by selvagsz** [abuiles/ember-watson#70]
Ran `methodify` command on a test repo but throws the following error

`app/routes/application.js - Line 3: Unexpected identifier`

```javascript
// routes/application.js
import Ember from 'ember';

const { get, set } = Ember;

export default Ember.Route.extend({
  model: function() {
    //some stuff
  },

  setupController: function(controller, context) {
    //some extra stuffs
    this._super(controller, context);
  }
});
```

Related to #30

**Comment by marcioj** [comment]
Hi @selvagsz thanks for report!

Probably this is a recast issue due to `get` and `set` be keywords

```js
var recast = require('recast');
recast.parse('const { get, set } = Ember;'); // Error: Line 1: Unexpected identifier
recast.parse('const { foo, bar } = Ember;'); // { ast object }
```

I'll dig more into this tonight

**Comment by marcioj** [comment]
Fixed in `0.6.3`. Thanks for open the issue!

**Comment by selvagsz** [comment]
:+1:
